Change logger_mode flag to be actually interpreted as an octal (#7273)
osquery/osquery: docs/wiki/installation/cli-flags.md

Directory path for `ERROR`/`WARN`/`INFO` and query result logging by the **filesystem** plugin.

`--logger_mode=0640`

> File mode for output log files by the **filesystem** plugin (provided as an octal string). Note that this affects both the query result log and the status logs and only works on POSIX platforms.
> **Warning**: If run as root, log files may contain sensitive information! Also, versions previous to osquery 5.0.0 were actually interpreting it as a number in decimal format, not octal.

Reviewer: ```suggestion
File mode for output log files by the **filesystem** plugin, provided as an octal string. Note that this affects both the query result log and the status logs and only works on POSIX platforms. (Versions previous to osquery 5.0.0 were incorrectly interpreting `logger_mode` as a number in decimal format, not octal.)
**Warning**: If run as root, log files may contain sensitive information! 
```
Maybe?

Author: Oups, yeah the "it" in the sentence was also out of place there!
